persistence-jdbc/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcPersistenceModuleSpec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence.jdbc
+
+import javax.inject.Inject
+import org.scalatest._
+import play.api.PlayException
+import play.api.db.DBApi
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.inject.{ bind => playBind }
+
+/**
+ * This test will simply wire a minimal application using Guice.
+ *
+ * It tests that Lagom correctly overrides Play's configuration to
+ * fail fast when database is not available.
+ */
+class JdbcPersistenceModuleSpec
+  extends AsyncWordSpec
+  with Matchers
+  with BeforeAndAfterAll {
+
+  "The JdbcPersistenceModule" should {
```

Comment:
Adapted from the work @renatocaval did here:

https://github.com/lagom/lagom/pull/1409/files#diff-d837e416e2bbba189f663af9dd38253aR30